docs/examples/CSI/csi_example.md

```diff
@@ -57,6 +57,8 @@
 
 ### Create a StorageClass and VolumeShapshotClass:
 
+- Exisiting VolumeSnapshotClass definitions can not be modified, a new VolumeSnapshotClass must be created.
```

Comment:
Only true for ones managed by an operator/controller.. which is most of cloud provider defaults

Reply:
thanks, updated!